pyro/contrib/README.md

```diff
@@ -0,0 +1,12 @@
+# Contributed Code
```

Comment:
For similar organization, see https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib